### PR TITLE
[Priest] Fae Guardians

### PR DESCRIPTION
**Pull request by TJMoats**
Shadow Priest:

Screen+Shot+2020-12-28+at+4 08 31+PM

Holy / Disc:

Screen+Shot+2020-12-28+at+4 09 05+PM

### CODE REVIEW

File: src/parser/priest/holy/modules/Abilities.ts [hunk 1]
```diff
@@ -395,6 +395,18 @@ class Abilities extends CoreAbilities {
...
         },
       },
+      {
+        spell: SPELLS.FAE_GUARDIANS,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        cooldown: 90,
+        enabled: combatant.hasCovenant(COVENANTS.NIGHT_FAE.id),
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.8,
```
Comment (Pewtro) [inline src/parser/priest/holy/modules/Abilities.ts]: no GCD?

**Comment by Pewtro** [comment]
Looks fine outside of my one comment, for future reference you *could* if you wanted to add it to abilities file through the FaeGuardian module itself.

**Comment by TJMoats** [comment]
I had no idea that I could add it in the module. I've done so now. Thanks for the comment :D

**Comment by Pewtro** [comment]
Thanks!

**Pull request merged**